notebooks/Understand_Tables.ipynb

```diff
@@ -0,0 +1,4387 @@
+{
```

Comment:
<p>So this extra space was due to a bug in Watson Table Understanding?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Technically it's a bug in the PDF-&gt;HTML conversion upstream of Watson Table Understanding. Most likely the PDF document kerned the "i" closer to the "f" than to the "c", and the conversion software mistook the difference in spacing as a space after the "c".</p>